src/settings-ui/Settings.UI/Flyout/LaunchPage.xaml

```diff
@@ -10,157 +10,137 @@
-                                </controls:FlyoutMenuButton.Icon>
-                            </controls:FlyoutMenuButton>
-                        </DataTemplate>
-                    </ItemsControl.ItemTemplate>
-                </ItemsControl>
-            </Grid>
+                            FontSize="12"
+                            Glyph="&#xe76c;" />
+                    </StackPanel>
+                </Button.Content>
+            </Button>
         </Grid>
-        <Grid Grid.Row="2">
+        <ItemsControl
+            Grid.Row="1"
+            Padding="12,0"
+            HorizontalAlignment="Center"
+            VerticalAlignment="Top"
+            ItemsSource="{x:Bind ViewModel.FlyoutMenuItems}"
+            TabNavigation="Local">
+            <ItemsControl.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <toolkitControls:WrapPanel VerticalSpacing="12" />
+                </ItemsPanelTemplate>
+            </ItemsControl.ItemsPanel>
+            <ItemsControl.ItemTemplate>
+                <DataTemplate x:DataType="viewModels:FlyoutMenuItem">
+                    <controls:FlyoutMenuButton
+                        Click="ModuleButton_Click"
+                        Tag="{x:Bind Tag}"
+                        AutomationProperties.Name="{x:Bind Label}"
+                        ToolTipService.ToolTip="{x:Bind ToolTip}"
+                        Visibility="{x:Bind Visible, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged, Converter={StaticResource BoolToVisibilityConverter}}">
+                        <controls:FlyoutMenuButton.Content>
+                            <TextBlock
+                                Style="{StaticResource CaptionTextBlockStyle}"
+                                Text="{x:Bind Label}"
+                                TextAlignment="Center"
+                                TextWrapping="Wrap" />
+                        </controls:FlyoutMenuButton.Content>
+                        <controls:FlyoutMenuButton.Icon>
+                            <Image>
+                                <Image.Source>
+                                    <BitmapImage UriSource="{x:Bind Icon, Mode=OneWay}" />
+                                </Image.Source>
+                            </Image>
+                        </controls:FlyoutMenuButton.Icon>
+                    </controls:FlyoutMenuButton>
+                </DataTemplate>
+            </ItemsControl.ItemTemplate>
+        </ItemsControl>
+        <!-- bottom buttons -->
+        <StackPanel
+            Grid.Row="2"
+            Padding="12,0">
             <InfoBar
                 x:Uid="UpdateAvailable"
                 IsClosable="False"
                 IsOpen="{x:Bind ViewModel.IsUpdateAvailable, Mode=OneWay}"
-                Severity="Success" />
+                Severity="Informational" />
 
-            <StackPanel
-                Grid.Row="1"
-                Margin="0,0,12,0"
+            <CommandBar
```

Comment:
Specific reason on why to use a CommandBar here? The current designs of the buttons where by design spec.. We are just displaying 3 buttons here, so using a CommandBar brings little value - and loads more XAML into the visual tree?

Reply:
Also, the icons are pretty big now.. and spacing is incorrect. See:

<img width="619" alt="image" src="https://user-images.githubusercontent.com/9866362/226325614-6f36c893-10b1-4b7f-aa8d-db0fa18c1a45.png">

Reply:
I think perceived size is relative. On a 1920×1080 screen it's a "presence", but on a 4k screen or something like that (which I don't have) I suspect it looks fine. Also think about touch screens, where a bit of size/padding is welcome.